app/enterprise/1.5.x/support-policy.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Kong Enterprise Version Support Policy
+---
+
+
+The Support Policy for Kong Enterprise software versions is explained in this topic. 
+
+For customers with an active enterprise support agreement, Kong provides full support for each release for 12 months from the released date. 
+
+Full support includes:
+
+* Technical support on the use of the software as documented, assistance with application configuration, and guidelines on performance tuning. 
+
+* Access to all error corrections, bug fixes, software updates and software upgrades to the Software that Kong makes generally available to other Kong enterprise customers who have purchased a license to the same Software purchased by Customer (“Upgrades”). 
```

Comment:
```suggestion
* Access to all error corrections, bug fixes, software updates, and software upgrades to the Software that Kong makes generally available to other Kong enterprise customers who have purchased a license to the same Software purchased by Customer (“Upgrades”). 
```
Editing to add comma. I also don't think you need to saying "software" before "updates" or "upgrades", but that might be legalese, so I left it alone.